weblate/formats/ttkit.py

```diff
@@ -506,7 +506,7 @@ def all_store_units(self):
         )
 
     def delete_unit(self, ttkit_unit) -> Optional[str]:
-        self.store.removeunit(ttkit_unit)
+        return self.store.removeunit(ttkit_unit)
```

Comment:
This is wrong, this method intentionally returns None. The problem with that is that adding `return None` is pointless :-). Anyway this is not related to topic of this PR.